### PR TITLE
change import function to fread to improve import of tabular

### PR DESCRIPTION
**Pull request by sbenateau**
some space have been removed automatically by my text editor, I hope it doesn't change anything.

### CODE REVIEW

File: tools/regionalgam/flight-curve.R [hunk 1]
```diff
@@ -1,7 +1,12 @@
 #!/usr/bin/env Rscript
 args = commandArgs(trailingOnly=TRUE)
 source(args[1]) #TODO replace by library(regionalGAM) if available as official package from bioconda
-tryCatch({input = read.table(args[2], header=TRUE,sep="	")},finally={input = read.table(args[2], header=TRUE,sep=",")})
+library(data.table)
+# import file
+input = fread(args[2])
+# retransform to data.frame to allow the calculation by the fonctions from the package
```
Comment (bgruening) [inline tools/regionalgam/flight-curve.R]: functions?

File: tools/regionalgam/ab-index.R [hunk 2]
```diff
@@ -4,8 +4,10 @@ args = commandArgs(trailingOnly=TRUE)
 source(args[1])
 
 
-tryCatch({input = read.table(args[2], header=TRUE,sep=" ")},finally={input = read.table(args[2], header=TRUE,sep=",")})
-pheno = read.table(args[3], header=TRUE,sep="	")
+# import file
+input = fread(args[2])
+# retransform to data.frame to allow the calculation by the fonctions from the package
```
Comment (bgruening) [inline tools/regionalgam/ab-index.R]: functions

File: tools/regionalgam/ab-index.R [hunk 3]
```diff
@@ -4,8 +4,10 @@ args = commandArgs(trailingOnly=TRUE)
 source(args[1])
 
 
-tryCatch({input = read.table(args[2], header=TRUE,sep=" ")},finally={input = read.table(args[2], header=TRUE,sep=",")})
-pheno = read.table(args[3], header=TRUE,sep="	")
```
Comment (bgruening) [inline tools/regionalgam/ab-index.R]: what happend to pheno?

File: tools/regionalgam/ab-index.R (hunk 2 above)
Comment (sbenateau) [reply tools/regionalgam/ab-index.R]: error from my side ! Thank you @bgruening

File: tools/regionalgam/flight-curve.R (hunk 1 above)
Comment (sbenateau) [reply tools/regionalgam/flight-curve.R]: the functions that are in the file dennis-gam-initial-function.R

File: tools/regionalgam/ab-index.R (hunk 3 above)
Comment (sbenateau) [reply tools/regionalgam/ab-index.R]: error from my side ! Thank you @bgruening

File: tools/regionalgam/ab-index.xml [hunk 4]
```diff
@@ -3,12 +3,15 @@
     <macros>
         <import>regionalgam_macros.xml</import>
     </macros>
+    <requirements>
+        <requirement type="package" version="1.12.2">r-data.table</requirement>
+    </requirements>
     <expand macro="rg_r_requirements"/>
```
Comment (Alanamosse) [inline tools/regionalgam/ab-index.xml]: Try to remove this macro line

Comment (sbenateau) [reply tools/regionalgam/ab-index.xml]: Thank you @Alanamosse  !

**Comment by bgruening** [comment]
Can we get a rebase here?

**Comment by sbenateau** [comment]
Actually it doesn't make so much sense anymore...

**Comment by sbenateau** [comment]
Actually it doesn't make so much sense anymore...

**Pull request closed without merging**